{
  "modified": "2024-10-16T14:52:32Z",
  "published": "2024-10-16T14:52:32Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "MAL-2024-6109"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "testdontdownloadthis"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/testdontdownloadthis/MAL-2024-6109.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "1935d918b522bf7e6a4f84d0900768dd2e9a221b082f46535b2b8d8e4927a708",
        "import_time": "2024-10-24T01:00:18.370273949Z",
        "id": "RLUA-2024-09382",
        "modified_time": "2024-10-16T14:52:32Z"
      }
    ]
  }
}
